src/cli.js

```diff
@@ -15,25 +15,31 @@ import initDocker from './docker/docker-create';
 import ciChoices from '../ci-choices.json';
 import containerizationChoices from '../containerization-choices.json';
 
-
-function writeFiles(config) {
+/**
+ * Initializes the files and accounts needed to use or application
+ * @param config the config object form the main inquirer prompt
+ */
+function initProject(config) {
+  // if the project doesn't already exist, intialize the files and accounts
   if (!fs.existsSync(config.projectName)) {
     fs.mkdirSync(config.projectName);
     fs.mkdirSync(`${config.projectName}/src`);
     createPackageJson(config);
     createIndexFile(config.projectName);
-  }
-  if (config.container === 'Docker') {
-    initDocker(config);
-  }
-  if (config.ci === 'TravisCI') {
-    initTravisCI(config);
+
+    if (config.container === 'Docker') {
+      initDocker(config);
+    }
+    if (config.ci === 'TravisCI') {
+      initTravisCI(config);
+    }
```

Comment:
Not sure who wrote this, but this should run initDocker and initTravisCi at about the same time. This could mess up the log, since the logs will not be in sequence. If that isn't a problem, this is fine. Otherwise, this should be made synchronous.

Reply:
Won't they happen asynchronously, they both use fs.writeFile()?

Reply:
Yes, since they are async, they may execute concurrently, and may not be in order. So the logs will not be in order.

Reply:
I don't think this is something that we should worry about. Neither of the processes are dependent on one another.